app/src/forms/rbac/routes.js

```diff
@@ -9,42 +9,578 @@ const { currentUser, hasFormPermissions, hasSubmissionPermissions, hasFormRoles,
 
 routes.use(currentUser);
 
+/**
+ * @openapi
+ * /rbac/current:
+ *  get:
+ *    tags:
+ *      - RBAC
+ *    description: This endpoint will get list of users and roles for a form
+ *    security:
+ *      - bearerAuth: []
+ *      - openId: []
+ *    responses:
+ *      '200':
+ *        description: Sucess
+ *        content:
+ *          application/json:
+ *            schema:
+ *              $ref: '#/components/responses/responseBody/RBACGetCurrentUser'
+ *      '403':
+ *        $ref: '#/components/responses/Error/AccessDenied'
+ *      '5XX':
+ *        $ref: '#/components/responses/Error/UnExpected'
+ */
+
 routes.get('/current', keycloak.protect(), async (req, res, next) => {
   await controller.getCurrentUser(req, res, next);
 });
 
+/**
+ * @openapi
+ * /rbac/current/submissions:
+ *  get:
+ *    tags:
+ *      - RBAC
+ *    description: ⁠This endpoint will fetch all the submissions in chefs for the current user. This list also includes each submission's user permissions, roles, and submission status.
+ *    security:
+ *      - bearerAuth: []
+ *      - openId: []
+ *    responses:
+ *      '200':
+ *        description: Success
+ *        content:
+ *          application/json:
+ *           schema:
+ *               $ref: '#/components/responses/responseBody/RBACGetCurrentUserSubmissions'
+ *      '403':
+ *        $ref: '#/components/responses/Error/AccessDenied'
+ *      '5XX':
+ *        $ref: '#/components/responses/Error/UnExpected'
+ */
 routes.get('/current/submissions', keycloak.protect(), async (req, res, next) => {
   await controller.getCurrentUserSubmissions(req, res, next);
 });
 
+/**
+ * @openapi
+ * /rbac/idps:
+ *  get:
+ *    tags:
+ *      - RBAC
+ *    description: This endpoint will get list of a list of identity providers.
+ *    security:
+ *      - bearerAuth: []
+ *      - openId: []
+ *    responses:
+ *      '200':
+ *        description: Sucess
```

Comment:
Typo here: "Sucess" instead of "Success".

Reply:
Fixed

Reply:
Fixed